js/screens/SignIn/SignInContainer.js

```diff
@@ -13,9 +15,117 @@ export default class SignInContainer extends Component {
   });
   constructor() {
```

Comment:
pass props in your constructor and your super if you are using props in your components.